app/models.py

```diff
@@ -22,3 +22,12 @@ class Message(db.Model):
     send_time_unix = db.Column(db.Text())
     subject = db.Column(db.Text())
     thread_id = db.Column(db.Text())
+
+
+class Media(db.Model):
+    __tablename__ = "media"
+
+    id = db.Column(db.Integer, primary_key=True)
+    media_type = db.Column(db.Text())
+    sender_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    url = db.Column(db.Text())
```

Comment:
Looking at what we want in the API response, we'll need to add timestamp and recipient_id into here.

or... do you think it makes more sense to have
- media_type 
- url
- message_id (db.ForeignKey('messages.id'))

maybe if we know we're going to be querying specifically for media, we're okay with denormalizing the data so we can get at it quickly instead of having to reference the message table for all other information?